packages/@aws-cdk/aws-apigatewayv2/lib/websocket/route.ts

```diff
@@ -37,6 +37,13 @@ export interface WebSocketRouteOptions {
    * @default - No Authorizer
    */
   readonly authorizer?: IWebSocketRouteAuthorizer;
+
+  /**
+   * Should the route send a response to the client
+   * @default false
+   */
+  readonly shouldReturnResponse?: boolean;
```

Comment:
0.02$: is the `should` not superfluous?

Reply:
do you think it should be named only `returnResponse`?